File: semaine4/NuancierRVBGeek_OC/NuancierRVBGeek_OC/ViewController.m
//
//  ViewController.m
//  NuancierRVBGeek_OC
//
//  Created by Cao Sang DOAN on 03/10/15.
//  Copyright © 2015 Cao Sang DOAN. All rights reserved.
//

#import "ViewController.h"
#import "MyView.h"

@interface ViewController ()

@end

@implementation ViewController

MyView *myView;

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    UIScreen *screen = [UIScreen mainScreen];
    CGRect frame = [screen bounds];
    myView = [[MyView alloc] initWithFrame:frame];
    [myView setBackgroundColor:UIColor.whiteColor];
    [self setView:myView];
    [myView release];
    
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (BOOL)shouldAutorotate
{
    return YES;
}

- (BOOL) shouldAutorotateToInterfaceOrientation:(UIInterfaceOrientation)toInterfaceOrientation
{
    return YES;
}

- (void) viewWillTransitionToSize:(CGSize)size withTransitionCoordinator:(id<UIViewControllerTransitionCoordinator>)coordinator
{
    [myView viewWillTransitionToSize:size];
}

@end

File: semaine5_imagier_nav/Imagier_Swift/Imagier_Swift/ViewController.swift
//
//  ViewController.swift
//  Imagier_Swift
//
//  Created by Cao Sang DOAN on 10/10/15.
//  Copyright (c) 2015 Cao Sang DOAN. All rights reserved.
//

import UIKit

class ViewController: UIViewController{

    private var myView: MyView!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        let screen = UIScreen.mainScreen()
        myView = MyView(frame: screen.bounds)
        
        self.view = myView
        
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    override func viewWillTransitionToSize(size: CGSize, withTransitionCoordinator coordinator: UIViewControllerTransitionCoordinator) {
        myView.viewWillTransitionToSize(size)
    }


}

